app/exporters/shopify/variant_exporter.rb

```diff
@@ -0,0 +1,43 @@
+module Shopify
+  class VariantExporter
+    include Spree::Retail::PresenterHelper
+
+    def initialize(spree_variant_id:, variant_klass: Spree::Variant,
+                   variant_api: ShopifyAPI::Variant,
+                   variant_converter: Shopify::VariantConverter)
+
+      @spree_variant = variant_klass.find(spree_variant_id)
+      @variant_converter = variant_converter
+      @variant_api = variant_api
+    end
+
+    def perform
+      shopify_variant = find_shopify_variant_for(spree_variant)
+      shopify_variant.update_attributes(variant_attributes)
+      save_pos_variant_id(spree_variant, shopify_variant)
+
+      shopify_variant
```

Comment:
It really helps for specs to return the object. Instead of having to re-ping the api to fetch it. What do you think?